docs/quality-principles/introduction.md

```diff
@@ -0,0 +1,44 @@
+# Dasharo Quality Principles
+
+## Secure Delivery
+
+Each Dasharo release is secured cryptographically, ensuring with RSA 4096 keys,
+authenticity of binary SHA-256 integrity checksum. The three-step key
+verification assuming that your client receives a clean file without any
+third-party modifications. Integrity and signature verification procedure is
+included so that the key compliance can be easily checked.
+
+## Secure Maintenance
+
+Dasharo doesn't believe in security through obscurity. That is why each Dasharo
+firmware is supported with the system of Transparent Validation, proving its
+efficiency and quality with testing results. As an Open Source product, it
+provides fast emergency releases and patches together with long-lasting
```

Comment:
provides **fast** emergency --> change to quick or timely?

Reply:
changed to timely